settings.gradle

```diff
@@ -1,7 +1,6 @@
 include ':vector'
 
-
 // uncomment theses lines to compile matrix SDK as project sources instead of external lib
 // see build.gradle section Matrix SDK management
 //include ':matrix-sdk'
-//project(':matrix-sdk').projectDir = new File(settingsDir, '../matrix-android-sdk/matrix-sdk')
+//project(':matrix-sdk').projectDir = new File(settingsDir, '../matrix-android-sdk/matrix-sdk')
```

Comment:
Why this file has been modified? Please revert

Reply:
Only one empty line is deleted in re-formatting. No content has been changed

Reply:
ok